src/api/app/models/bs_request_action.rb

```diff
@@ -786,6 +786,18 @@ def check_for_expand_errors!(add_revision)
       query[:expand] = 1 unless updatelink
       query[:rev] = source_rev if source_rev
       dir = Xmlhash.parse(Backend::Api::Sources::Package.files(source_project, source_package, query))
+      unless add_revision
+        # Enforce revisions?
+        tprj = Project.get_by_name(target_project)
+        add_revision = tprj.instance_of?(Project && tprj.find_attribute('OBS', 'EnforceRevisionsInRequests').present?
```

Comment:
This is why RuboCop is barking:

```suggestion
        add_revision = tprj.instance_of?(Project) && tprj.find_attribute('OBS', 'EnforceRevisionsInRequests').present?
```

Reply:
fixed that a while ago. And another rubocop complain now.